### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by pyhys**

### CODE REVIEW

**Review by olbjo — APPROVED** [review]
Great!
No problems found.

Future additions could be:
- Add device specific alarms to gsdml file.
- Table on tests could be updated to have several comlums. Name; Time; Resets; B required; PLC required.. depending on table formatting possibilities
   
-

File: doc/compliancetest.rst [hunk 1]
```diff
@@ -123,6 +123,8 @@ Project > Run.
 
 When communication is verified, enable all relevant test cases.
 
+The ART tester tool is storing Wireshark files (.pcap files) in the
```
Comment (hefloryd) [inline doc/compliancetest.rst:126]: is storing -> stores

File: doc/compliancetest.rst [hunk 2]
```diff
@@ -306,15 +308,15 @@ Relevant test cases for conformance class A
 +-------------------------------------------------+---------------------------------------------------------------+
 | Different Access Ways                           | Requires additional hardware ("Device B")                     |
 +-------------------------------------------------+---------------------------------------------------------------+
-| PDEV_CHECK_ONEPORT                              | Requires additional hardware ("Device B"). Power cycle twice. |
+| PDEV_CHECK_ONEPORT                              | Requires additional hardware ("Device B"). Power cycle x3     |
```
Comment (hefloryd) [inline doc/compliancetest.rst:311]: 3 times?

File: doc/compliancetest.rst [hunk 3]
```diff
@@ -391,22 +394,43 @@ Set the ``AddressAssignment`` attribute to ``DHCP``.
...
+It is allowed that 3-6 data frames are sent before sending the alarm frame.
+
+Perform the cable unplugging measurements with reduction ratios 1, 2, 4, 8 and 16.
+The data hold time should be 3x the frame interval.
+With a cycle time of 1 ms this corresponds to a frame send interval of
+1 ms to 16 ms, and a data hold time of 3 ms to 48 ms.
+
+In Siemens TIA portal, set the update time to for example 2 ms (values
+1, 2, 4, 8, and 16 ms should be used).
+The "Acceped update cycles without IO data" should be set to 3.
```
Comment (hefloryd) [inline doc/compliancetest.rst:422]: Accepted

File: doc/compliancetest.rst [hunk 4]
```diff
@@ -391,22 +394,43 @@ Set the ``AddressAssignment`` attribute to ``DHCP``.
...
+
+At startup the first valid data frame should be sent withing the data hold time.
```
Comment (hefloryd) [inline doc/compliancetest.rst:424]: within

File: doc/compliancetest.rst [hunk 5]
```diff
@@ -418,5 +442,7 @@ Run with PLC for 10 minutes without errors. Record startup and data exchange usi
 
 Interoperability with controller
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Run with PLC, then switch PLC to stop. Study the outputs of the IO-device.
-Disconnect cable from PLC. Study the outputs of the IO-device.
+Run with PLC, and verify that the outputs are according to manual of your IO-device.
```
Comment (hefloryd) [inline doc/compliancetest.rst:445]: to the manual

File: doc/compliancetest.rst (hunk 1 above)
Comment (pyhys) [reply doc/compliancetest.rst]: Done

File: doc/compliancetest.rst (hunk 2 above)
Comment (pyhys) [reply doc/compliancetest.rst]: Done

File: doc/compliancetest.rst (hunk 3 above)
Comment (pyhys) [reply doc/compliancetest.rst]: Done

File: doc/compliancetest.rst (hunk 4 above)
Comment (pyhys) [reply doc/compliancetest.rst]: Done

File: doc/compliancetest.rst (hunk 5 above)
Comment (pyhys) [reply doc/compliancetest.rst]: Done

**Pull request merged**